Content.Server/Cluwne/CluwneSystem.cs

```diff
@@ -0,0 +1,120 @@
+using Content.Server.Administration.Commands;
+using Content.Server.Speech.Components;
+using Content.Server.Popups;
+using Content.Shared.Popups;
+using Content.Server.Interaction.Components;
+using Content.Server.Speech;
+using Content.Shared.Mobs;
+using Robust.Shared.Random;
+using Robust.Shared.Audio;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Stunnable;
+
+namespace Content.Server.Cluwne;
+
+public sealed class CluwneSystem : EntitySystem
+{
+    [Dependency] private readonly PopupSystem _popupSystem = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly VocalSystem _vocal = default!;
+    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+    [Dependency] private readonly SharedStunSystem _stunSystem = default!;
+
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<CluwneComponent, MobStateChangedEvent>(OnMobState);
+        SubscribeLocalEvent<CluwneComponent, ComponentStartup>(OnComponentStartup);
+    }
+
+    private void OnMobState(EntityUid uid, CluwneComponent component, MobStateChangedEvent args)
+    {
+        // removes cluwne component if cluwne dies.
+        if (args.NewMobState == MobState.Dead)
+           RemComp<CluwneComponent>(uid);
+
+    }
+
+    private void OnComponentStartup(EntityUid uid, CluwneComponent component, ComponentStartup args)
+    {
+        var meta = MetaData(uid);
+        var name = Name(uid);
+
+        //accent, it is backwards but with some thought you can still communicate.
+        EnsureComp<BackwardsAccentComponent>(uid);
+
+        var vocal = EnsureComp<VocalComponent>(uid);
+        var scream = new SoundCollectionSpecifier("CluwneScreams");
+        vocal.FemaleScream = scream;
+        vocal.MaleScream = scream;
+
+        //popup x has turned into a cluwne and play bikehorn when cluwnified.
+        _popupSystem.PopupEntity(Loc.GetString("cluwne-transform", ("target", uid)), uid, PopupType.LargeCaution);
+        _audio.PlayPvs(component.SpawnSound, uid);
+
+        //gives it the funny "Cluwnified ___" name.
+        meta.EntityName = Loc.GetString("cluwne-name-prefix", ("target", name));
+
+        //gives the cluwne costume and makes cluwne clumsy.
+        SetOutfitCommand.SetOutfit(uid, "CluwneGear", EntityManager, (_, clothing) =>
+        {
+            EnsureComp<ClumsyComponent>(uid);
+        });
```

Comment:
this runs for every piece of clothing, the clumsy ensure should come out of the loop and this should be the same unremoveable thing the smite did:
```suggestion
        //gives the cluwne costume and makes cluwne clumsy.
        EnsureComp<ClumsyComponent>(uid);
        SetOutfitCommand.SetOutfit(uid, "CluwneGear", EntityManager, (_, clothing) =>
        {
            if (HasComp<ClothingComponent>(clothing))
                EnsureComp<UnremoveableComponent>(clothing);
        });
```

Reply:
(probably need to import that too but w/e)

Reply:
i removed the unremoveable from the system cause i dont want all the items in cluwnegear to be unremoveable just certain ones so i added it to those specific entities only.

Reply:
i moved clumsy out and removed the loop. The unremovable is on the entities for the clothing since the lcuwne clothing has no other purpose than to be unremovable so i didnt add it to the system.

Reply:
alright